setup/pi/configure.sh

```diff
@@ -261,6 +299,31 @@ function install_matrix_packages () {
   pip3_install matrix-nio
 }
 
+function install_tesla_ble_packages () {
+  local install_path="$1"
+  local binary_dir=/tmp/binarydir
+
+  apt install -y pi-bluetooth bluez
```

Comment:
pi-bluetooth won't exist on all distros, and the make-root-fs-readonly.sh script deletes bluez, which will also delete pi-bluetooth again because it depends on bluez.

Reply:
Okay, looks like `pi-bluetooth` is specific to Pi 3 anyway. The `bluetooth` package looks more generic and picks up `bluez` as a dependency automatically.

As for the removal in `make-root-fs-readonly.sh`, should we change that globally or attempt to make it conditional? (I'm a little surprised it hasn't removed it on my system. It looks like it's run on every upgrade?)